README.rst

```diff
@@ -57,7 +53,20 @@ To install in a virtual environment in your current project:
 Usage Example
 =============
 
-.. todo:: Add a quick, simple example. It and other examples should live in the examples folder and be included in docs/examples.rst.
+.. code-block:: python
+	import board
+	import adafruit_touchscreen
+
+	# These pins are used as both analog and digital! XR and YU must be analog
+	# and digital capable. XL and YD just need to be digital
+	ts = adafruit_touchscreen.Touchscreen(board.TOUCH_XL, board.TOUCH_XR,
```

Comment:
Just noticed this, it should be updated to require X-RAY be analog as well

Reply:
fixed